config/jobs/kubernetes/sig-storage/sig-storage-gce-config.yaml

```diff
@@ -1,8 +1,61 @@
 presubmits:
   kubernetes/kubernetes:
+  - name: pull-kubernetes-e2e-gce-storage-slow
+    always_run: false
+    optional: true
+    # pkg/controller/volume
+    # pkg/kubelet/volumemanager
+    # pkg/volume
+    # pkg/util/mount
+    # test/e2e/storage
+    # test/e2e/testing-manifests/storage-csi
+    run_if_changed: '^(pkg\/controller\/volume|pkg\/kubelet\/volumemanager|pkg\/volume|pkg\/util\/mount|test\/e2e\/storage|test\/e2e\/testing-manifests\/storage-csi)'
```

Comment:
This doesn't change it to required right?

Reply:
I think it'll make it automatically run when those files change, but it will still be optional in the sense that it can be skipped even if it's failing.

Reply:
this was the other thing I was hung up on, I think if you trigger these you would want them to be blocking, and they aren't specified as such now

that I don't know this off the top of my head means I'm inclined to drop a /hold and let you decide via `/hold cancel` if you'd like to just move ahead as is